lib/matplotlib/_pylab_helpers.py

```diff
@@ -84,6 +84,7 @@ def destroy_fig(cls, fig):
 
     @classmethod
     def destroy_all(cls):
+        import gc
```

Comment:
To clarify, this is *only* in this method since it also is registered as an `atexit` function.  Otherwise it wouldn't be necessary here.